src/transport-zeromq/receiver.cpp

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
Going to pull out these changes and apply it on `develop` then rebase this in.
A cleaner header is nicer for folding.